solara/components/markdown.py

```diff
@@ -157,6 +158,11 @@ def _markdown_template(
         async loadMermaid() {
             return (await this.import([`${this.getCdn()}/mermaid@10.8.0/dist/mermaid.min.js`]))[0]
         },
+        async loadMHchem() {
+            await this.loadKatex();
```

Comment:
```suggestion
            await this.loadKatex();
```
```suggestion
```

This will cause the CSS to be added twice,